mkdocs.yml

```diff
@@ -75,6 +75,7 @@ nav:
         - TLS Communication: Concepts/TLS.md
     - Reference:
         - EthSigner command line: Reference/CLI/CLI-Syntax.md
+        - EthSigner API Methods: Reference/API-Methods.md
```

Comment:
This is also something I introduced but I think we can drop EthSigner on this line and the previous one - it's implied by the fact it's the EthSigner docs.